packages/fx-core/src/common/samples.ts

```diff
@@ -41,7 +41,7 @@ class SampleProvider {
             url:
               (sample as any).relativePath && (sample as any).url
                 ? (sample as any).url
-                : sampleConfigV3.baseUrl,
```

Comment:
Should L60 be changed as well?

Reply:
Will we have other hotfix release for v4? We can update if yes.

Reply:
I see. As long as both v3 and v2 work, we can leave it alone.

Reply:
Actually hotfix release for v2 will be shipped from last hotfix branch instead of dev branch. So this won't affect v2.